### ISSUE TITLE
tf2onnx.convert ValueError: Value "<dtype: 'int32'>" is not valid attribute data type.

### ISSUE THREAD

**Issue by mnboos** [onnx/tensorflow-onnx#362]
```
root@fbb1f76a6c32:/data# python3 -m tf2onnx.convert --input /output_dir/model.pb --inputs Placeholder --outputs gradients/bidirectional_rnn/bw/bw/while/Switch_4_grad/b_switch --output /output_dir/model.onnx --fold_const --verbose --opset 8 --continue_on_err
using tensorflow=1.11.0, onnx=1.4.1, opset=8, tfonnx=0.4.0/bd38f6
2019-03-11 06:46:32.313323: I tensorflow/tools/graph_transforms/transform_graph.cc:318] Applying fold_constants
2019-03-11 06:46:32.376268: E tensorflow/tools/graph_transforms/transform_graph.cc:333] fold_constants: Ignoring error You must feed a value for placeholder tensor 'cnnKernel_4' with dtype float and shape [3,3,128,256]
         [[{{node cnnKernel_4}} = Placeholder[dtype=DT_FLOAT, shape=[3,3,128,256]]()]]
2019-03-11 06:46:32.391037: I tensorflow/tools/graph_transforms/transform_graph.cc:318] Applying remove_attribute
2019-03-11 06:46:32.399893: I tensorflow/tools/graph_transforms/transform_graph.cc:318] Applying fold_batch_norms
2019-03-11 06:46:32.426097: I tensorflow/tools/graph_transforms/transform_graph.cc:318] Applying fold_old_batch_norms
2019-03-11 06:46:32.631110: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2
ERROR:tf2onnx:pass1 convert failed for name: "gradients/bidirectional_rnn/bw/bw/while/TensorArrayWrite/TensorArrayWriteV3_grad/TensorArrayReadV3/f_acc"
op: "StackV2"
input: "gradients/bidirectional_rnn/bw/bw/while/TensorArrayWrite/TensorArrayWriteV3_grad/TensorArrayReadV3/Const"
attr {
  key: "elem_type"
  value {
    type: DT_INT32
  }
}
attr {
  key: "stack_name"
  value {
    s: ""
  }
}
, ex=Value "<dtype: 'int32'>" is not valid attribute data type.
Traceback (most recent call last):
  File "/usr/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/usr/local/lib/python3.5/dist-packages/tf2onnx-0.4.0-py3.5.egg/tf2onnx/convert.py", line 123, in <module>
  File "/usr/local/lib/python3.5/dist-packages/tf2onnx-0.4.0-py3.5.egg/tf2onnx/convert.py", line 105, in main
  File "/usr/local/lib/python3.5/dist-packages/tf2onnx-0.4.0-py3.5.egg/tf2onnx/tfonnx.py", line 2472, in process_tf_graph
  File "/usr/local/lib/python3.5/dist-packages/tf2onnx-0.4.0-py3.5.egg/tf2onnx/tfonnx.py", line 144, in tensorflow_to_onnx
  File "/usr/local/lib/python3.5/dist-packages/tf2onnx-0.4.0-py3.5.egg/tf2onnx/tfonnx.py", line 131, in tflist_to_onnx
  File "/usr/local/lib/python3.5/dist-packages/onnx-1.4.1-py3.5-linux-x86_64.egg/onnx/helper.py", line 56, in make_node
    for key, value in sorted(kwargs.items()))
  File "/usr/local/lib/python3.5/dist-packages/onnx-1.4.1-py3.5-linux-x86_64.egg/onnx/helper.py", line 56, in <genexpr>
    for key, value in sorted(kwargs.items()))
  File "/usr/local/lib/python3.5/dist-packages/onnx-1.4.1-py3.5-linux-x86_64.egg/onnx/helper.py", line 255, in make_attribute
    'Value "{}" is not valid attribute data type.'.format(value))
ValueError: Value "<dtype: 'int32'>" is not valid attribute data type.
```

**Comment by nbcsm** [comment]
thanks for reporting.
is that possible to share your model pb file please?
or a faked model with simple tf scripts that can reproduce this issue?

**Comment by mnboos** [comment]
I can share an untrained version of the model.
[model.pb.gz](https://github.com/onnx/tensorflow-onnx/files/2951097/model.pb.gz)

**Comment by nbcsm** [comment]
thanks, we will take a look and get back to you later.

**Comment by mnboos** [comment]
Thanks a lot!

**Comment by VigneshRavichandran** [comment]
I'm facing a similar issue. Is this resolved? Please let me know

**Comment by lucienwang1009** [comment]
@mnboos the shared model cannot be imported into tensorflow due to float_ref input of ApplyRMSProp op. It seems the graph wasn't frozen appropriately.
@VigneshRavichandran Did you use the latest converter? Could you please share your model?

**Comment by guschmue** [comment]
tfonnx=0.4.0/bd38f6 ... that is pretty old. Do a 
```pip install -U tf2onnx``` 
Newer versions of tf2onnx will try to freeze the graph for you.

**Comment by VigneshRavichandran** [comment]
Hi,

Tried with the new version 1.5.1. Still facing the same issue

**Comment by GLBurnett** [comment]
I'm having this same issue in version 1.5.4

**Comment by jignparm** [comment]
@GLBurnett , can you paste a link to the model (or past it directly if it's small), and also the command line you are using to convert?

**Comment by woaixiaojiaa** [comment]
@guschmue  @jignparm   hello, i also met similar question , ValueError: Value "<dtype: 'int64'>" is not valid attribute data type 
is there helps? thank you !!!

**Comment by guschmue** [comment]
Don't know what op this is but it looks like something with sparse tensors which we don't support. The reason why we don't support it is that there is no runtime that supports it.

**Comment by woaixiaojiaa** [comment]
> Don't know what op this is but it looks like something with sparse tensors which we don't support. The reason why we don't support it is that there is no runtime that supports it.

sorry,i do not understand "there is no runtime" ? could you please say it in detail? or are there any solutions?

**Comment by guschmue** [comment]
sparse_type in the error message could be a sparse tensor thing or tf.parse_example.
For sparse tensors -  onnx spec does support this to some degree but as far I know there is no runtime that implements that.
If your error comes from tf.parse_example, there is no onnx spec support for this.

Important to know would be the op that triggered the error.

**Comment by guschmue** [comment]
assuming this means sparse support. This would need some onnx changes as well as support by onnxruntime before we can do something on this.

**Comment by eddiecong** [comment]
Any updates on this? Facing the same problem.

**Comment by TomWildenhain-Microsoft** [comment]
Hi @eddiecong, can you open a new issue?  The error message here is pretty generic and can come from any op we haven't supported yet.  Also, try using the latest tf2onnx from github, since it should fix this error message and give a more understandable one.  Thanks.

`pip uninstall tf2onnx`
`pip install git+https://github.com/onnx/tensorflow-onnx`
